pilot/cmd/pilot-agent/options/options.go

```diff
@@ -78,6 +78,8 @@ var (
 		"The grace period ratio for the cert rotation, by default 0.5.").Get()
 	workloadRSAKeySizeEnv = env.Register("WORKLOAD_RSA_KEY_SIZE", 2048,
 		"Specify the RSA key size to use for workload certificates.").Get()
+	isDualUseEnv = env.Register("IS_DUAL_USE", false,
```

Comment:
I don't think this should be a setting in agent - the CA can decide what to include ( and should filter anything else )